docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,185 @@ tags:
 
 # Cedarling Quick Start Guide
 
-## This content is in progress
+## Introduction
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+In this guide, we'll demonstrate how to use [Jans Tarp](../../demos/jans-tarp) to register a client on an authorization server, and use the embedded cedarling webassembly app to reach an authorization decision. This will be done in three steps:
```

Comment:
You need some more background here... something like: Jans Tarp is a browser plugin that enables developers to test OpenID Connect flows. It embeds the Cedarling WASM PDP, and is one of the fastest ways to test out Cedar with real JWT tokens.